src/components/ChartContainer/D3Container.js

```diff
@@ -62,7 +63,19 @@ function D3Container({ dashboardState, dashboardActions, store }) {
   }));
 
   useEffect(() => {
-    setDisplayData(store.results.map((result) => ({ ...result })));
+    function handleResize() {
+      setGraphWidth(window.innerWidth)
+    }
+
+    window.addEventListener('resize', handleResize)
+
+    return () => {
+      window.removeEventListener('resize', handleResize)
```

Comment:
What's the documentation on this?

Reply:
Here is a website that describes the resizing logic with a bit more detail.

https://www.pluralsight.com/guides/re-render-react-component-on-window-resize#module-cleanuplisteners